package.json

```diff
@@ -23,19 +23,27 @@
     "lint:js": "eslint .",
     "start": "ember serve",
     "test": "npm-run-all lint:* test:*",
+    "test:node": "qunit node-tests",
     "test:ember": "ember test",
     "test:ember-compatibility": "ember try:each",
     "prepublishOnly": "ember ts:precompile",
     "postpublish": "ember ts:clean"
   },
   "dependencies": {
     "axe-core": "^4.0.1",
+    "body-parser": "^1.19.0",
     "broccoli-funnel": "^3.0.3",
     "broccoli-merge-trees": "^4.2.0",
+    "date-and-time": "^0.14.1",
     "ember-cli-babel": "^7.21.0",
     "ember-cli-typescript": "^4.0.0",
     "ember-cli-version-checker": "^5.1.1",
-    "ember-destroyable-polyfill": "^2.0.1"
+    "ember-destroyable-polyfill": "^2.0.1",
+    "express": "^4.17.1",
+    "fs-extra": "^9.0.1",
+    "node-fetch": "^2.6.0",
+    "qunit": "^2.11.0",
+    "tmp": "^0.2.1"
```

Comment:
I think these might need to be devDependencies?

Reply:
Yep, I was avoiding the `no-unpublished-require` lint violation, but as you pointed out just adding that to `.npmignore` is the right path here.